### PR TITLE
Make after party col span 6

### PR DESCRIPTION
**Pull request by shiraji**
## Issue
#219 

## Overview (Required)
* Add new session type `DINNER`
* After party cell col span should be 6 

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionViewModel.java [hunk 1]
```diff
@@ -117,6 +117,8 @@ private int decideColSpan(@NonNull Session session, int roomCount) {
             return 3;
         } else if (session.isBreak()) {
             return roomCount;
+        } else if (session.isDinner()) {
+            return 6;
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionViewModel.java]: This is better 😃 

```java
} else if (session.isBreak() || session.isDinner()) {
    return roomCount;
```

Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionViewModel.java]: Sorry, my explanation was lacked 🙇

Comment (shiraji) [inline app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionViewModel.java]: Does `roomCount` always 6? If so, we could change `Type` to the following code.

```java
private enum Type {
        CEREMONY(3), SESSION(1), BREAK(6), DINNER(6);

        int colspan;

        Type(int colspan) {
            this.colspan = colspan;
        }

        @Override
        public String toString() {
            return super.toString().toLowerCase();
        }
    }
```

and then remove `isBreak()` etc methods, and these if/else statements.

Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionViewModel.java]: `roomCount` is always 6 on DroidKaigi2017. But it's calculated by sessions data.

And in my opinion, Session.Type should not have UI data like colSpan.
It's better to switch colspan in ViewModel.
What do you think?

Comment (shiraji) [inline app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionViewModel.java]: OK now got it. I think `Session.Type` is weird. It should not be there.

Comment (shiraji) [inline app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionViewModel.java]: never mind. let me sleep tonight. I will think about this one.

By the way, I totally agree with what you wrote. colspan should be handled in ViewModel.

Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionViewModel.java]: Thank you! Sorry, coflict occurred. Can you merge master tomorrow? 🙇

**Comment by shiraji** [comment]
done + rebase the commits.

After spent one night, I still cannot figure out the best practice...so I fixed the if/else statements. If I found something later, I will send other PR.

**Comment by konifar** [comment]
Thanks! Yes, I don't think this is the best way too. Anyway, I appreciate your work!
LGTM 👍

**Pull request merged**